test/unit/clarity_ext/helpers.py

```diff
@@ -0,0 +1,33 @@
+from clarity_ext.domain import Container, Analyte
+from mock import MagicMock
+
+
+def fake_analyte(container_id, artifact_id, sample_id, artifact_name, well, udfs):
+    """
+    Creates a fake analyte
+
+    TODO: Consider setting the domain objects only (not the rest resources) and
+    create tests for the mapping only
+    """
+    container_resource_mock = MagicMock(name="Container")
```

Comment:
Mock the domain objects only, but create a test for the factory methods, showing that the mapping logic between the two works as expected.